### DISCUSSION TITLE
CloudlogCAT with Omnirig V2.1

### DISCUSSION THREAD

**Discussion by PA4RB** [magicbug/Cloudlog#1325] [Q&A] [answered]
Anyone any luck on using Omnirig V2.1 (http://www.hb9ryz.ch/omnirig/) with CloudlogCAT?

**Comment by AndreasK79** [comment]
As far as I can find out, you need to use this version for V2.1 https://github.com/W3AXL/CloudlogCATv2

**Reply by PA4RB** [reply]
Hmmmm this one is not working on my system. Looking further....

**Comment by magicbug** [comment, marked as answer]
Never tried it with v2 to be able to comment on it I suspect it would need some code changes

So for now with my version I suspect the answer is no it wont

**Reply by AndreasK79** [reply]
Is says on Omnirig 2.1 page that V1 is not compatible with V2. The fork I linked to, says it's updated to work with 2.1. I got it to work locally with 2.1 as well (and found a bug I need to fix).

**Reply by PA4RB** [reply]
Validation error on installing this on Windows 11.  Any chance of helping me out here? Thanks in advance.

**Reply by AndreasK79** [reply]
@PD0RZH not sure I can help. If you can post a picture of the error, we might be able to help. I downloaded the source and ran it via Visual Studio, so I didn't have that problem.

**Reply by PA4RB** [reply]
@AndreasK79 I now see that WSJT-X for example is not (yet) compatible with Omnirig V2. I'll leave it for now. Thanks for offering the help though.
